src/gravel/controllers/orch/ceph.py

```diff
@@ -108,17 +111,17 @@ def call(self, cmd: Dict[str, Any]) -> Any:
 
 class Mgr(Ceph):
 
-    def __init__(self):
-        super().__init__()
+    def __init__(self, conf_file: str = CEPH_CONF_FILE):
```

Comment:
fyi, mons and mgrs don't use the `"/etc/ceph/ceph.conf"` at all. The file is just there for admin's convenience

Reply:
eh but these are mon clients via rados .. unless I'm misunderstanding what you mean?